src/Components/ExercisePage/Solution.tsx

```diff
@@ -1,3 +1,38 @@
+import { FileArrowUp, Plus } from 'phosphor-react';
+import { useRef } from 'react';
+
 export const Solution = () => {
-  return <p>Solution</p>;
+  const hiddenFileInput = useRef<HTMLInputElement>(null);
+
+  const handleClick = () => {
+    if (hiddenFileInput.current === null) return undefined;
```

Comment:
Are you sure that current is exactly null and cannot be undefined? If not please use "==" instead of "===". Also I think the if-statement should not be on a single line and should always be encapsulated with brackets.

Reply:
Maybe use "!"?